src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallsController.kt

```diff
@@ -92,6 +96,25 @@ class RecallsController(
       )
     )
   }
+
+  @PostMapping("/recalls/{recallId}/assignee")
```

Comment:
Think we agreed (just this morning) to add `{assigneeId}` as ending path param rather than body - for consistency with delete?